sdks/python/apache_beam/runners/common.py

```diff
@@ -546,17 +563,23 @@ def process_outputs(self, windowed_input_element, results):
 class _OutputProcessor(OutputProcessor):
   """Processes output produced by DoFn method invocations."""
 
-  def __init__(self, window_fn, main_receivers, tagged_receivers):
+  def __init__(self,
+               window_fn,
+               main_receivers,
+               tagged_receivers,
+               per_element_output_counter):
     """Initializes ``_OutputProcessor``.
 
     Args:
       window_fn: a windowing function (WindowFn).
       main_receivers: a dict of tag name to Receiver objects.
       tagged_receivers: main receiver object.
+      per_element_output_counter: per_element_output_counter of one work_item
```

Comment:
<!--new_thread; commit:c89749a0d77350dfbd2686db975aa087bafb8f1b; resolved:0-->
Please add comment that this can be None to disable this output counter.